source/patterns/@aws-solutions-constructs/aws-cloudfront-s3/lib/index.ts

```diff
@@ -143,6 +150,98 @@ export class CloudFrontToS3 extends Construct {
     this.cloudFrontWebDistribution = cloudFrontDistributionForS3Response.distribution;
     this.cloudFrontFunction = cloudFrontDistributionForS3Response.cloudfrontFunction;
     this.cloudFrontLoggingBucket = cloudFrontDistributionForS3Response.loggingBucket;
-  }
 
+    // Define the OriginAccessControl
+    const originAccessControl = new cloudfront.CfnOriginAccessControl(this, 'CloudFrontOac', {
+      originAccessControlConfig: {
+        name: `cloudfront-default-oac-${new Date().getTime().toString(16)}`,
```

Comment:
Fixed this by using `generatePhysicalName('aws-cloudfront-s3', ['oac'], 16)` to create a unique name. Tested  and answered the following questions before settling on this approach:

- Does not specifying anything for `name` tell CloudFormation to give it a unique name? Didn't get that far, turns out its a required property by CDK, so that didn't work.
- Does giving the OAC a general name like `cloudfront-s3-oac` tell CloudFormation to still make it unique? For example, `cloudfront-s3-oac-stackNameHere-someOtherRandomValues`? No, it's a literal property specification, so if you provide `name: cloudfront-s3-oac`, that will be the name. No adjustments or prefixes/suffixes added by CFN. This caused issues when running integration tests because one or more OACs were named the same, and the same issue would pop up if a customer had greater than one deployment of this construct.